wasm2c/wasm-rt-impl.c

```diff
@@ -281,31 +280,49 @@ void wasm_rt_init(void) {
 }
 
 bool wasm_rt_is_initialized(void) {
-#if WASM_RT_MEMCHECK_SIGNAL_HANDLER && !WASM_RT_SKIP_SIGNAL_RECOVERY
+#if WASM_RT_INSTALL_SIGNAL_HANDLER
   return g_signal_handler_installed;
 #else
   return true;
 #endif
 }
 
 void wasm_rt_free(void) {
-#if WASM_RT_MEMCHECK_SIGNAL_HANDLER && !WASM_RT_SKIP_SIGNAL_RECOVERY
+#if WASM_RT_INSTALL_SIGNAL_HANDLER
   os_cleanup_signal_handler();
 #endif
 }
 
+#if WASM_RT_MEMORY_ALLOCATION_MMAP
```

Comment:
Likelywise what do you think of `WASM_RT_USE_MMAP`?   This read like the kind of cmake/autoconf configuration macro that I grew up on so maybe I'm biased?